CHANGELOG.md

```diff
@@ -1,5 +1,128 @@
+
+TODO INSERT HIGHLIGHTS
+
+### `@material-ui/core@`
+
+- <!-- 01 --> [BottomNavigation] Migrate to emotion (#24556) @vinyldarkscratch
+- <!-- 03 --> [CardActions] Fix wrong classes export name (#24609) @mnajdova
+- <!-- 04 --> [Card] Migrate CardContent to emotion (#24600) @vicasas
+- <!-- 05 --> [Card] Migrate to emotion (#24597) @povilass
+- <!-- 06 --> [TextField] Migrate InputBase to emotion (#24555) @duganbrett
+- <!-- 07 --> [Accordion] Allow to disable gutter/spacing (#24532) @TimonPllkrn
+- <!-- 08 --> [List] Migrate to emotion (#24560) @vinyldarkscratch
+- <!-- 09 --> [Card] Migrate CardActions to emotion (#24604) @vicasas
+- <!-- 11 --> [List] Migrate ListSubheader to emotion (#24561) @vinyldarkscratch
+- <!-- 12 --> [Breadcrumbs] Migrate to emotion (#24522) @vinyldarkscratch
+- <!-- 13 --> [Divider] Migrate to emotion (#24558) @vinyldarkscratch
+- <!-- 14 --> [Switch] Migrate SwitchBase to emotion (#24552) @vinyldarkscratch
+- <!-- 15 --> [Hidden] Migrate to emotion (#24544) @vinyldarkscratch
+- <!-- 16 --> [List] Migrate ListItem to emotion (#24543) @xs9627
+- <!-- 18 --> [TextField] Fix Google Translate zero-width space issue (#24563) @d3mp
+- <!-- 19 --> [Table] Separate classes for different labels (#24568) @tonysepia
+- <!-- 22 --> [Accordion] Migrate AccordionSummary to emotion (#24540) @vinyldarkscratch
+- <!-- 23 --> [IconButton] Migrate to emotion (#24542) @vinyldarkscratch
+- <!-- 26 --> [Accordion] Migrate AccordionActions to emotion (#24538) @vinyldarkscratch
+- <!-- 27 --> [Accordion] Migrate AccordionDetails to emotion (#24539) @vinyldarkscratch
+- <!-- 30 --> [Link] Migrate to emotion (#24529) @praveenkumar-kalidass
+- <!-- 31 --> [Accordion] Migrate to emotion (#24518) @vinyldarkscratch
+- <!-- 34 --> [Backdrop] Migrate to emotion (#24523) @vinyldarkscratch
+- <!-- 41 --> [Grid] Add system props (#24499) @mnajdova
+- <!-- 42 --> [Icon] Migrate to emotion (#24516) @vinyldarkscratch
+- <!-- 44 --> [Collapse] Migrate to emotion (#24501) @vinyldarkscratch
+- <!-- 47 --> [SvgIcon] Migrate to emotion (#24506) @oliviertassinari
+- <!-- 48 --> [Avatar] Migrate AvatarGroup to emotion (#24452) @praveenkumar-kalidass
+- <!-- 49 --> [Box] Add back system props (#24485) @mnajdova
+- <!-- 50 --> [Alert] Migrate AlertTitle to emotion (#24448) @povilass
+- <!-- 54 --> [Alert] Migrate to emotion (#24442) @kutnickclose
+- <!-- 59 --> [l10n] Improve Hebrew translation (#24449) @eladmotola
+- <!-- 61 --> [Checkbox][Switch] Document defaultChecked (#24446) @praveenkumar-kalidass
+- <!-- 62 --> [AppBar] Migrate to emotion (#24439) @povilass
+- <!-- 64 --> [l10n] Improve German translation (#24436) @lukaselmer
+- <!-- 65 --> [Button][Badge] Support custom colors and sizes (#24408) @mnajdova
+- <!-- 70 --> [Grid] Fix horizontal scrollbar and nested dimensions (#24332) @greguintow
+- <!-- 73 --> [Grid] Migrate to emotion (#24395) @mnajdova
+- <!-- 74 --> [Badge] Fix TS custom variants (#24407) @mnajdova
+
+### `@material-ui/lab@`
```

Comment:
What about the versions? Once we hit a stable release the lab won't match. There is also the types package that isn't following the same versioning

Reply:
my todo :grind:

Reply:
done